app/models/mixins/live_metrics_mixin.rb

```diff
@@ -19,7 +24,7 @@ def first_and_last_capture(interval_name = "realtime")
       end.transpose
       adjust_timestamps(firsts, lasts, interval_name)
     rescue => e
-      _log.error("Hawkular service unavailable: #{e.message}")
+      _log.error("LiveMetrics unavailable: #{e.message}")
```

Comment:
LiveMetrics started as something specific for Hawkular, but the LiveMetric model and LiveMetricMixin is generic, so these two models have not specific info related the provider.
Another provider can reuse it.
That's main reason of this change.